### DISCUSSION TITLE
Configure KotlinSerializationConfig

### DISCUSSION THREAD

**Discussion by pwlan** [quarkusio/quarkus#25702] [Q&A]
Hi,

I am just trying to find out how to configure the Kotlin Serialization extension. 
I am using Quarkus 2.9.0.Final and the Rest Client with @RegisterRestClient.
I have tried 
```
quarkus.kotlin-serialization.json.ignoreUnknownKeys=true
quarkus.json.ignoreUnknownKeys=true
quarkus.kotlin-serialization.ignoreUnknownKeys=true
kotlin-serialization.json.ignoreUnknownKeys=true
```
but nothing seems to work. 
Grateful for any help.
Thanks.

[a comment by an automated account is not shown]

**Comment by med01-flutter-uki** [comment]
For anybody else struggling with configuring the Kotlin Serialization extension, I found the following worked (note kebab-case rather than camelCase):

```
quarkus.kotlin-serialization.json.ignore-unknown-keys=true
```
